operator-framework/src/test/java/io/javaoperatorsdk/operator/WorkflowActivationConditionIT.java

```diff
@@ -0,0 +1,50 @@
+package io.javaoperatorsdk.operator;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+import io.javaoperatorsdk.operator.junit.LocallyRunOperatorExtension;
+import io.javaoperatorsdk.operator.sample.workflowactivationcondition.WorkflowActivationConditionCustomResource;
+import io.javaoperatorsdk.operator.sample.workflowactivationcondition.WorkflowActivationConditionReconciler;
+import io.javaoperatorsdk.operator.sample.workflowactivationcondition.WorkflowActivationConditionSpec;
+
+import static io.javaoperatorsdk.operator.sample.workflowactivationcondition.ConfigMapDependentResource.DATA_KEY;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class WorkflowActivationConditionIT {
+
+  public static final String TEST_RESOURCE_NAME = "test1";
+  public static final String TEST_DATA = "test data";
+
+  @RegisterExtension
+  LocallyRunOperatorExtension extension =
+      LocallyRunOperatorExtension.builder()
+          .withReconciler(WorkflowActivationConditionReconciler.class)
+          .build();
+
+  @Test
+  void routeIsNotCreated() {
```

Comment:
renamed the method since, but we can check if the actual platform not supporting routes. But as we discussed before, IMO the platform is our case is give (based on definition of E2E tests in github actions) that it is pure k8s (minikube) not openshift. But we can double check in test, so if that would change we would be notified.